src/ng1/directives/staticTooltip/staticTooltip.controller.js

```diff
@@ -206,6 +206,10 @@ export default function StaticTooltipCtrl($scope, $element, $parse, safeAnimatio
     function(){ 
         $staticTooltip.destroyAllTooltips();
     });
+
+    $scope.$on('$destroy', function() {
+        $staticTooltip.destroyAllTooltips();
```

Comment:
This should probably only destroy the tooltip bound to this scope, we probably shouldn't remove all tooltips whenever the scope of one is destroyed. I may be wrong, but check to see if only destroying the tooltip bound to that scope works